### PR TITLE
Rename spawn_data_pipeline.py to spawn_pipeline.py

### LINKED ISSUE #490: Rename spawn_data_pipeline.py to spawn_pipeline.py
Since it supports spawning both data and check (or any other pipeline defined in a yaml file).

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #490

### CODE REVIEW

File: bugbug/model.py [hunk 1]
```diff
@@ -229,7 +229,7 @@ def get_top_features(cutoff, shap_values):
         return important_features
 
     def print_feature_importances(
-        self, important_features, feature_names, class_probabilities=None
+        self, important_features, class_probabilities=None
```
Comment (marco-c) [inline bugbug/model.py]: The changes in this file are unrelated, please revert them.

File: scripts/bug_classifier.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def classify_bugs(model_name, classifier):
             feature_names = model.get_human_readable_feature_names()
 
             model.print_feature_importances(
-                importance["importances"], feature_names, class_probabilities=probas
+                importance["importances"], class_probabilities=probas
```
Comment (marco-c) [inline scripts/bug_classifier.py]: Here too

**Comment by Patil2099** [comment]
@marco-c  Fixed

**Comment by marco-c** [comment]
> @marco-c Fixed

You're actually still modifying those files :P

**Comment by Patil2099** [comment]
@marco-c It's Fixed Finally
